client.lua

```diff
@@ -882,13 +882,15 @@ CreateThread(function() -- Speeding
             local ped = PlayerPedId()
             if IsPedInAnyVehicle(ped, false) then
                 local speed = GetEntitySpeed(GetVehiclePedIsIn(ped, false)) * speedMultiplier
-                local stressSpeed = seatbeltOn and config.MinimumSpeed or config.MinimumSpeedUnbuckled
+		local vehicle = GetVehiclePedIsIn(ped, false)
+		local vehicleClass = GetVehicleClass(vehicle)
```

Comment:
Please format both lines correctly so they align with the rest of the code

Reply:
I updated by pasting white space from line 884. Did that resolve the issue?

Reply:
It did